src/main/java/com/crowdin/client/ai/AiApi.java

```diff
@@ -0,0 +1,196 @@
+package com.crowdin.client.ai;
+
+import com.crowdin.client.ai.model.AddAiPromptRequest;
+import com.crowdin.client.ai.model.AddAiProviderRequest;
+import com.crowdin.client.ai.model.AiPrompt;
+import com.crowdin.client.ai.model.AiPromptResponseList;
+import com.crowdin.client.ai.model.AiPromptResponseObject;
+import com.crowdin.client.ai.model.AiProvider;
+import com.crowdin.client.ai.model.AiProviderModel;
+import com.crowdin.client.ai.model.AiProviderModelResponseList;
+import com.crowdin.client.ai.model.AiProviderResponseList;
+import com.crowdin.client.ai.model.AiProviderResponseObject;
+import com.crowdin.client.ai.model.AiProxyChatCompletionRequest;
+import com.crowdin.client.ai.model.AiProxyChatCompletionResponse;
+import com.crowdin.client.core.CrowdinApi;
+import com.crowdin.client.core.http.HttpRequestConfig;
+import com.crowdin.client.core.http.exceptions.HttpBadRequestException;
+import com.crowdin.client.core.http.exceptions.HttpException;
+import com.crowdin.client.core.model.ClientConfig;
+import com.crowdin.client.core.model.Credentials;
+import com.crowdin.client.core.model.PatchRequest;
+import com.crowdin.client.core.model.ResponseList;
+import com.crowdin.client.core.model.ResponseObject;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class AiApi extends CrowdinApi {
+
+    public AiApi(Credentials credentials) {
+        super(credentials);
+    }
+
+    public AiApi(Credentials credentials, ClientConfig clientConfig) {
+        super(credentials, clientConfig);
+    }
+
+    /**
+     * @param userId User identifier
+     * @return list of Ai Prompts
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.ai.prompts.getMany" target="_blank"><b>API Documentation</b></a></li>
+     * </ul>
+     */
+    public ResponseList<AiPrompt> listAiPrompts (Long userId, Long projectId, String action, Long limit, Integer offset) throws HttpException, HttpBadRequestException {
```

Comment:
```suggestion
    public ResponseList<AiPrompt> listAiPrompts(Long userId, Long projectId, String action, Long limit, Integer offset) throws HttpException, HttpBadRequestException {
```

Usually we don't put a space after the method name. Please fix this here and in other methods